src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -299,6 +321,14 @@ private static Assembly LoadAssembly(string assemblyName, bool throwOnFail)
             Assembly assembly = null;
             string path = Path.GetFullPath(assemblyName);
             assembly = Assembly.LoadFile(path);
+            if(assembly == null)
+            {
+                if (throwOnFail)
```

Comment:
The method has one caller only. And that caller always set `throwOnFail` to true. We can remove the argument.